caffe2/operators/mean_op.h

```diff
@@ -72,6 +83,34 @@ class MeanOp final : public Operator<Context> {
           Input(0).dtype().name());
     }
   }
+
+ private:
+  std::vector<int64_t> ComputeBroadcastDims(
```

Comment:
@BIT-silence do we have some existing code which can be reused in this case?

Reply:
While @BIT-silence responds, one option I thought about was to use `elementwise_ops_util::ComputeBinaryBroadcastForwardDims` one dimension at a time (broadcasting Input i and the current Output obtained form Inputs 0 to i-1). 

Would that be preferable?

Reply:
I think it's fine to make another function from the efficiency side. Since it will also be used for SumOp, I'm wondering if we can make it as another util function. But it's fine to just keep it here.

Reply:
Ok, let's keep the current solution. Later we can merge them together.